tree-sitter-stack-graphs/src/loader.rs

```diff
@@ -325,17 +329,28 @@ impl Loader {
         graph: &mut StackGraph,
         cancellation_flag: &dyn CancellationFlag,
     ) -> Result<(), LoadError<'a>> {
-        let file = graph.add_file(&path.to_string_lossy()).unwrap();
+        let file_name = path.to_string_lossy();
+        let file: stack_graphs::arena::Handle<stack_graphs::graph::File> =
+            graph.add_file(&file_name).unwrap();
+        let builtins_root = Path::new("");
         let mut globals = Variables::new();
         Self::load_globals_from_config_str(&config, &mut globals)?;
```

Comment:
We could add to `globals` here as well, but I think it is nicer to just require those variables to be set in `builtins.cfg`. See for example:

https://github.com/github/stack-graphs/blob/558d1da3c27dfe50a45cc9acbee66bc5d898d314/languages/tree-sitter-stack-graphs-typescript/src/builtins.cfg.